Cargo.toml

```diff
@@ -193,9 +193,7 @@ rpath = false
 
 [patch.crates-io]
 # If there are dependencies that need patching, they can be listed below.
-# For example:
-# arrow-format = { git = "https://github.com/datafuse-extras/arrow-format", rev = "78dacc1" }
 
-arrow2 = { git = "https://github.com/jorgecarleitao/arrow2", rev = "33f6ba1" }
+arrow-format = { git = "https://github.com/sundy-li/arrow-format", rev = "77da470" }
```

Comment:
Maybe we can use https://github.com/datafuse-extras/arrow-format?

Reply:
It's trivial, in progress to replace arrow2 with arrow-rs.